deploy/009-deploy-vtokens.ts

```diff
@@ -20,7 +19,6 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deploy } = deployments;
   const { deployer } = await getNamedAccounts();
   const { tokensConfig, poolConfig, preconfiguredAddresses } = await getConfig(hre.network.name);
```

Comment:
We shouldn't need any preconfigured addresses. All addresses can come from the deployments included from the packages in the hardhat.config